EdgeOfEmpireBot/EdgeOfEmpireBot/Data/Commands.json

```diff
@@ -0,0 +1,9 @@
+{
+    "remember": "general",
+    "test": "general",
+    "add": "general",
+
+    "gr": "steam",
```

Comment:
'gr' and 'requestgame' are essentially the same command and just an alias of one other. 

Thoughts on modifying the structure of this json to include aliases, additionally also a flag to check if its actually enabled. 

```
{
    "commands": {
        "general": [
            {
                "command": "remember",
                "active": true
            },
            {
                "command": "test",
                "active": true
            },
            {
                "command": "add",
                "active": true
            }
        ],
        "steam": [
            {
                "command": "games",
                "active": true
            },
            {
                "command": "requestgame",
                "aliases": ["gr"],
                "active": true
            }
        ]
    }
}

```


We would need to modify the router code to reflect if we went down this path, but potentially nicer when adding commands especially if we have multiple thigns with same command, like 'roll' 
```
    if (!commands.TryGetValue(command!.ToLower(), out var process))
        {
            await messageService.SendMessage("[Error]: Invalid command detected...");
            return;
        }
```

Reply:
Have some thoughts on how we might do this, again messy initally but useful once made. 
Might make a seperate issue so this doesn't snowball ie build router then improve the router

Reply:
yeah I think for now this is a very barebones router. But I do think there is quite a lot of improvement that can be done.